zppy/templates/global_time_series.bash

```diff
@@ -17,64 +17,9 @@ cd {{ scriptDir }}
 echo "RUNNING ${id}" > {{ prefix }}.status
 
 # Generate global time series plots
-
-# Names
-moc_file={{ moc_file }}
-experiment_name={{ experiment_name }}
-figstr={{ figstr }}
-case={{ case }}
-
-# Years
-start_yr={{ year1 }}
-end_yr={{ year2 }}
-ts_num_years={{ ts_num_years }}
-
-# Paths
-www={{ www }}
-case_dir={{ output }}
-global_ts_dir={{ global_time_series_dir }}
-
 ################################################################################
 
-export CDMS_NO_MPI=true
-
-use_atm={{ use_atm }}
-use_lnd={{ use_lnd }}
-
-use_ocn={{ use_ocn }}
-if [[ ${use_ocn,,} == "true" ]]; then
-    echo 'Create ocean time series'
-    cd ${global_ts_dir}
-    mkdir -p ${case_dir}/post/ocn/glb/ts/monthly/${ts_num_years}yr
-    input={{ input }}/{{ input_subdir }}
-    python ocean_month.py ${input} ${case_dir} ${start_yr} ${end_yr} ${ts_num_years}
-    if [ $? != 0 ]; then
-      cd {{ scriptDir }}
-      echo 'ERROR (3)' > {{ prefix }}.status
-      exit 3
-    fi
-
-    echo 'Copy moc file'
-    cd ${case_dir}/post/analysis/mpas_analysis/cache/timeseries/moc
-    cp ${moc_file} ../../../../../ocn/glb/ts/monthly/${ts_num_years}yr/
-    if [ $? != 0 ]; then
-      cd {{ scriptDir }}
-      echo 'ERROR (5)' > {{ prefix }}.status
-      exit 5
-    fi
-
-fi
-
-echo 'Update time series figures'
-cd ${global_ts_dir}
-atmosphere_only={{ atmosphere_only }}
-python coupled_global.py ${case_dir} ${experiment_name} ${figstr} ${start_yr} ${end_yr} {{ color }} ${ts_num_years} {{ plots_original }} ${atmosphere_only,,} {{ plots_atm }} {{ plots_ice }} {{ plots_lnd }} {{ plots_ocn }} {{ regions }}
-if [ $? != 0 ]; then
-  cd {{ scriptDir }}
-  echo 'ERROR (6)' > {{ prefix }}.status
-  exit 6
-fi
-
+zppy-interfaces global-time-series --use_ocn={{ use_ocn }} --global_ts_dir={{ global_time_series_dir }} --input={{ input }} --input_subdir={{ input_subdir }}  --moc_file={{ moc_file }} --case_dir={{ output }} --experiment_name={{ experiment_name }} --figstr={{ figstr }} --color={{ color }} --ts_num_years={{ ts_num_years }} --plots_original={{ plots_original }} --atmosphere_only={{ atmosphere_only }} --plots_atm={{ plots_atm }} --plots_ice={{ plots_ice }} --plots_lnd={{ plots_lnd }} --plots_ocn={{ plots_ocn }} --regions={{ regions }} --start_yr={{ year1 }} --end_yr={{ year2 }}
```

Comment:
I like your idea of a config file instead of this.  It would be good to break the command into multiple lines with `\` for better readability if you do keep this approach.